docs/area-owners.md

```diff
@@ -74,9 +75,14 @@ Note: Editing this file doesn't update the mapping used by the `@msftbot` issue
+| area-System.Diagnostics-coreclr                | @tommcdon     | @tommcdon @Anipik @ViktorHofer                      | Sub-Areas: <ul><li>System.Diagnostics.EventLog - @Anipik @ViktorHofer</li></ul>                                                                                                                                                                                                           |
+| area-System.Diagnostics-mono                   | @lewing       | @thaystg  @radical                                  |                                                                                                                                                                                                                                                                                           |
 | area-System.Diagnostics.Activity               | @tommcdon     | @tarekgh                                            |                                                                                                                                                                                                                                                                                           |
+| area-System.Diagnostics.EventLog               | @jeffhandley  | @adamsitnik @carlossanlop @jozkee                   |                                                                                                                                                                                                                                                                                           |
+| area-System.Diagnostics.PerformanceCounter     | @jeffhandley  | @adamsitnik @carlossanlop @jozkee                   |                                                                                                                                                                                                                                                                                           |
 | area-System.Diagnostics.Process                | @jeffhandley  | @adamsitnik @carlossanlop @jozkee                   |                                                                                                                                                                                                                                                                                           |
 | area-System.Diagnostics.Tracing                | @tommcdon     | @noahfalk @tommcdon @Anipik @ViktorHofer @tarekgh   | Packages: <ul><li>System.Diagnostics.DiagnosticSource</li><li>System.Diagnostics.PerformanceCounter - @Anipik @ViktorHofer</li><li>System.Diagnostics.Tracing - @Anipik @ViktorHofer</li><li>System.Diagnostics.TraceSource</li></ul>                                                     |
```

Comment:
Looks like this is the only one that's still spanning teams.  Would it make sense to give DiagnosticSource its own label?
```suggestion
| area-System.Diagnostics.Tracing                | @tommcdon     | @noahfalk @tommcdon @Anipik @ViktorHofer @tarekgh   | Packages: <ul><li>System.Diagnostics.DiagnosticSource</li><li>System.Diagnostics.Tracing - @Anipik @ViktorHofer</li> 
```